blocks/api/registration.js

```diff
@@ -32,6 +32,7 @@ export function registerBlockType( slug, settings ) {
 		return;
 	}
 	if ( ! /^[a-z0-9-]+\/[a-z0-9-]+$/.test( slug ) ) {
+		console.log( slug );
```

Comment:
Ah, yeah, that could work. Minor, but I'd recommend adhering to JS standards even in the comments:

```js
/* eslint no-console: [ 'error', { allow: [ 'error' ] } ] */
```

Or we could change to `// eslint-disable-line` at the times we _want_ to allow `console`, depending on how noisy this becomes.